### DISCUSSION TITLE
Email notification to another in CVAT

### DISCUSSION THREAD

**Discussion by keshva23** [cvat-ai/cvat#4047] [Q&A] [answered]
How can i send email notification to another user in CVAT?

**Comment by bsekachev** [comment, marked as answer]
No way here.

**Reply by keshva23** [reply]
@bsekachev ok.Thank you.

**Reply by osinpl** [reply]
Hey!
Is there any way to configure CVAT to let users know by email notification that they are assigned to the specific task/job or project. Also if the supervisors and admins can have info about status change?

Br, Kris

**Reply by nmanovic** [reply]
@osinpl , at the moment the feature is not implemented

**Reply by osinpl** [reply]
is this somehow allocated in the road map 3,6,12 months? I'm asking because need to know is we should go into our own development of these features?

**Reply by nmanovic** [reply]
If you want to contribute the feature, please do. Please contact me on linkedin if you need any assistance.

**Reply by osinpl** [reply]
thank you! will talk with the team.

**Comment by osinpl** [comment]
Thanks for info!
